var/spack/repos/builtin/packages/py-entity-management/package.py

```diff
@@ -13,6 +13,7 @@ class PyEntityManagement(PythonPackage):
     git      = 'ssh://bbpcode.epfl.ch/nse/entity-management'
 
     version('1.2.10', tag='entity-management-v1.2.10')
+    version('0.1.12', tag='entity-management-v0.1.12', branch='v0')
```

Comment:
why `v0` branch?

Reply:
Versions of `entity-management` >= 0.1.9 and less than 1.0 are in the branch v0 (not master), so I needed to specify it.
I've added @genric to be sure that it's safe and correct.
Alternatively, we should update both `brainbuilder` and `subcellular-querier` to use `entity-management` >= 1.0, but I don't know if it's possible.

Reply:
Weird. Is that because Spack fetches a single branch by default?

Reply:
This is up to @tomdele to approve. I don't know which v0 nexus instance is used for that code. But technically, to make dependency checker pass, LGTM.

Reply:
> Weird. Is that because Spack fetches a single branch by default?

I probably did something wrong when trying before, I re-tried locally and it works without specifying the branch.
I'll remove the branch specification, thanks for looking at it.